core/src/main/java/tachyon/UnderFileSystem.java

```diff
@@ -62,6 +62,10 @@ public static UnderFileSystem get(String path) {
    * @return null for any unknown scheme.
    */
   public static UnderFileSystem get(String path, Object conf) {
+    if (path == null) {
+      return null;
```

Comment:
I refer to the *parse* function below, that it return null too. But I think your point is valid, maybe I should modify both?